rotkehlchen/exchanges/manager.py

```diff
@@ -112,22 +120,27 @@ def initialize_exchanges(
         log.debug('Initializing exchanges')
         # initialize exchanges for which we have keys and are not already initialized
         for name, credentials in exchange_credentials.items():
+            module_name = self._get_exchange_module_name(name)
             if name not in self.connected_exchanges:
                 try:
-                    module = import_module(f'rotkehlchen.exchanges.{name}')
+                    module = import_module(f'rotkehlchen.exchanges.{module_name}')
                 except ModuleNotFoundError:
                     # This should never happen
                     raise AssertionError(
                         f'Tried to initialize unknown exchange {name}. Should never happen.',
                     ) from None
 
-                exchange_ctor = getattr(module, name.capitalize())
+                exchange_ctor = getattr(module, module_name.capitalize())
                 extra_args: Dict[str, Any] = {}
                 if credentials.passphrase is not None:
                     extra_args['passphrase'] = credentials.passphrase
-                if name == 'kraken':
+                if name == str(Location.KRAKEN):
                     settings = database.get_settings()
                     extra_args['account_type'] = settings.kraken_account_type
+                if name == str(Location.BINANCE):
```

Comment:
here and below it should actually be an `elif`, since name can't be anything different and we don't return from the branch